build.sbt

```diff
@@ -9,7 +9,7 @@ libraryDependencies ++= Seq(
   "io.circe" %% "circe-core" % circeVersion,
   "io.circe" %% "circe-parser" % circeVersion,
   "org.scalaj" %% "scalaj-http" % "2.3.0",
-  "org.scalameta" %% "scalameta" % "2.1.2",
+  "org.scalameta" %% "scalameta" % "3.7.4",
```

Comment:
Wow, surprisingly simple. Good that we use only the stable API for code generation.